src/main/java/uk/gov/companieshouse/utils/RandomReferenceSequence.java

```diff
@@ -0,0 +1,15 @@
+package uk.gov.companieshouse.utils;
+
+import org.apache.commons.lang3.RandomStringUtils;
+
+public class RandomReferenceSequence {
+    public static String generate() {
+
+        StringBuilder builder = new StringBuilder();
+        return builder
+            .append(RandomStringUtils.random(5, true, true).toUpperCase())
+            .append('-')
+            .append(RandomStringUtils.random(5, true, true).toUpperCase())
+            .toString();
```

Comment:
This can be simplified to something like:
```suggestion
        String randomStr = RandomStringUtils.random(5, true, true).toUpperCase();
        return randomStr + '-' + randomStr;
```

Reply:
I'm not sure it can. Wouldn't it mean that the two strings on either side of the hyphen would be identical?

Reply:
hm, I dont think so, because randomStr will be assigned a value, and we want both sides of the '-' to be random

Reply:
Agreed that StringBuilder is overkill though. Its efficiency benefits are only realised in, say, a loop. Here, concatting the three strings together with `+` is likely (marginally) faster.

Reply:
I was not too bothered about efficiency for this case, I simply think its more readable

Reply:
But I can change it